lib/webauthn/attestation_statement/none.rb

```diff
@@ -6,7 +6,7 @@ module WebAuthn
   module AttestationStatement
     class None < Base
       def valid?(*_args)
-        true
+        [WebAuthn::AttestationStatement::Types::NONE, nil]
```

Comment:
I think with this new `Types` namespace I find reading this a bit confusing.

I am worried that reading "attestation statement type none" would confuse users into thinking this is talking about "attestation statement formats", which is a different thing.

I more inclined to do `WebAuthn::AttestationFormat::ATTESTATION_TYPE_NONE` or even `WebAuthn::ATTESTATION_TYPE_NONE`.

Reply:
I like `WebAuthn::AttestationFormat::ATTESTATION_TYPE_NONE`, I think it captures well how [Attestation Statement Formats part of the spec](https://www.w3.org/TR/webauthn/#attestation-formats) describes the relation between the two. Thanks, I've updated the PR!